src/Subsystems/Elevator.h

```diff
@@ -23,21 +23,41 @@ class Elevator: public Subsystem {
 private:
 	// It's desirable that everything possible is private except
 	// for methods that implement subsystem capabilities
-	// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
+
 	std::shared_ptr<DoubleSolenoid> haySqueeze;
 	std::shared_ptr<DigitalInput> rightHaySqueezeSensor;
 	std::shared_ptr<DigitalInput> leftHaySqueezeSensor;
 	std::shared_ptr<DoubleSolenoid> brake;
 	std::shared_ptr<CANTalon> leftSpool;
 	std::shared_ptr<CANTalon> rightSpool;
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=DECLARATIONS
 public:
+
+	enum BrakeValue {kBrake = DoubleSolenoid::kForward, kBrakeOff = DoubleSolenoid::kReverse };
+	enum HaySqueezeValue {kOpen = DoubleSolenoid::kForward, kClose = DoubleSolenoid::kReverse };
+
 	Elevator();
 	void InitDefaultCommand();
-	// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
 
-    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
+	//..........brake..........
+
+	void SetBrake(BrakeValue value);
+
+	bool IsBrakeOn();
+
+	//...........haysqueeze..........
+
+	bool IsHaySqueezeOpen();
```

Comment:
Should be `bool IsHaySqueezeOpen() const;`

Repeat for all `Is` functions or getters.